src/redux/books/books.js

```diff
@@ -1,45 +1,55 @@
-import { v4 as uuidv4 } from 'uuid';
+/* eslint-disable camelcase */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.